app/src/components/Chat/ChatList.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from "react";
 import { useAuth } from "../../AppProviders";
-import ProfilePopupModal from "../ProfilePopupModal";
+import ProfilePopupModal from "../UserProfile/ProfilePopupModal";
```

Comment:
I've renamed this to Profile, this might cause conflicts when we attempt to merge these

Reply:
Thanks for the heads up, will resolve it.